defaults/main.yml

```diff
@@ -83,6 +83,7 @@ wimpy_alarm_metrics:
 wimpy_deploy_strategy: "rolling_asg"
 
 wimpy_dns_ttl: 300 # DNS record Time To Live
+wimpy_new_dns_weight: 100 # DNS record weight
```

Comment:
100 is ok as value, but weight does not represent a percent value. This means that given the following scenario:
- DNS record A, weight: 100
- DNS record B, weight: 50

A will has 66.6% of traffic and B 33.3%. 
Percentage is calculated as PA = A / (A + B) * 100.

It's a matter of taste but I usually use lower values (from 0 to 10) because they allow good percentage ranges without working with very high values.